scene/gui/tab_bar.cpp

```diff
@@ -55,31 +63,34 @@ Size2 TabBar::get_minimum_size() const {
 			}
 		}
 
-		ms.width += Math::ceil(tabs[i].text_buf->get_size().x);
+		ms.width += tabs[i].size_text;
 		ms.height = MAX(ms.height, tabs[i].text_buf->get_size().y + y_margin);
 
+		Ref<StyleBox> style;
 		if (tabs[i].disabled) {
-			ms.width += tab_disabled->get_minimum_size().width;
+			style = tab_disabled;
 		} else if (current == i) {
-			ms.width += tab_selected->get_minimum_size().width;
+			style = tab_selected;
 		} else {
-			ms.width += tab_unselected->get_minimum_size().width;
+			style = tab_unselected;
 		}
 
+		ms.width = style->get_minimum_size().width;
```

Comment:
```suggestion
		ms.width += style->get_minimum_size().width;
```
I assume that would be the culprit for broken minimum size (untested).